policyengine_us/parameters/gov/states/nm/tax/income/credits/cdcc/divisor.yaml

```
@@ -0,0 +1,13 @@
+description: New Mexico divides the credit for dependent day care for seprate filers by this divisor.
+values:
+  2021-01-01: 2
+metadata:
+  unit: /1
+  label: New Mexico credit for dependent day care separate divisor
+  reference:
+    - title: New Mexico Administrative Code, § 3.3.13.9 - CREDIT FOR EXPENSES FOR DEPENDENT CHILD DAY CARE NECESSARY TO ENABLE GAINFUL EMPLOYMENT TO PREVENT INDIGENCY
+      href: https://www.law.cornell.edu/regulations/new-mexico/3-3-13-9-NMAC 
+    - title: 2022 New Mexico Income Tax Form Intructions
```

Since `#page=72` doesn't work with these. Also similarly to other files.
```suggestion
    - title: 2022 New Mexico Income Tax Form Instructions | Section V: New Mexico Child Day Care Credit (page 72)
```